packages/synthetix-main/contracts/modules/core/CollateralModule.sol

```diff
@@ -144,6 +144,28 @@ contract CollateralModule is
         return _getAccountUnassignedCollateral(accountId, collateralType);
     }
 
+    function depositMarketCollateral(
```

Comment:
these functions could be potentially moved to their own module. modules are, after all, public function interfaces to manipulate underlying pools of data